ac.soton.coda.simulator.ui/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,7 @@ Require-Bundle: org.eclipse.ui;bundle-version="[3.7.0,4.0.0)",
  org.eclipse.emf.ecore;bundle-version="[2.7.0,3.0.0)",
  org.eclipse.emf.transaction;bundle-version="[1.4.0,2.0.0)",
  org.eclipse.emf.workspace;bundle-version="[1.5.0,2.0.0)",
- org.eclipse.emf.ecore.xmi;bundle-version="[2.10.1,2.11.0)",
+ org.eclipse.emf.ecore.xmi;bundle-version="[2.12.0,3.0.0)",
```

Comment:
2.10 to 2.12?

Reply:
We could be conservative and set it to [2.10.1, 2.12.0] if you think it is necessary. Why it was set to be quite restrictive before?

Reply:
I think it is better to use [2.12-3.00) if we can.. especially if 2.12 is the Neon version. I don't know why it was more restricted before. Lets try to go with 2.12.